src/components/datatables/MUIDataTable.js

```diff
@@ -293,6 +295,10 @@ class MUIDataTable extends React.Component {
     this.initializeTable(this.props);
   }
 
+  componentWillReceiveProps() {
```

Comment:
@AjayDoddapaneni, Just FYI I have added this. Please let me know if I am missing any adverse impacts of this operation.